RawSpeed/DngDecoderSlices.cpp

```diff
@@ -143,6 +143,159 @@ my_error_throw (j_common_ptr cinfo)
   ThrowRDE("JPEG decoder error!");
 } 
 
+static void decodeFPDeltaRow(unsigned char * src, unsigned char * dst, size_t tileWidth, size_t realTileWidth, unsigned int bytesps, int factor) {
```

Comment:
From RawTherapee, dcraw.cc, it's license says:

   No license is required to download and use dcraw.c.  However,
   to lawfully redistribute dcraw, you must either (a) offer, at
   no extra charge, full source code* for all executable files
   containing RESTRICTED functions, (b) distribute this code under
   the GPL Version 2 or later, (c) remove all RESTRICTED functions,
   re-implement them, or copy them from an earlier, unrestricted
   Revision of dcraw.c, or (d) purchase a license from the author.

I chose option (c), since these functions are not restricted.